lxd/config/generate/lxddoc.go

```diff
@@ -191,7 +192,38 @@ func parse(path string, outputYAMLPath string, excludedPaths []string) (*doc, er
 		return nil, err
 	}
 
-	yamlDoc.Configs = projectEntries
+	// Within each group, alphabetically sort the entries by key (config option key)
+	orderedProjectEntries := make(map[string][]any, len(projectEntries))
+	for groupKey, entries := range projectEntries {
+		var sortedConfigOptionKeysPerGroup []string
+		for _, configEntry := range entries {
+			c := configEntry.(map[string]any)
+			for k := range c {
+				sortedConfigOptionKeysPerGroup = append(sortedConfigOptionKeysPerGroup, k)
+			}
+		}
+
+		sort.Strings(sortedConfigOptionKeysPerGroup)
+		for _, configOptionKey := range sortedConfigOptionKeysPerGroup {
+			for _, configEntry := range entries {
+				c := configEntry.(map[string]any)
+				for k := range c {
```

Comment:
```suggestion
				for k := range configEntry.(map[string]any) {
```

Reply:
Actually, `c` is further used within the loop

Reply:
Oh right, I overlooked that.